src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Data/EditFeatureLinkedAnnotation/EditFeatureLinkedAnnotation.xaml

```diff
@@ -0,0 +1,58 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.WPF.Samples.EditFeatureLinkedAnnotation.EditFeatureLinkedAnnotation"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" GeoViewTapped="MyMapView_GeoViewTapped" />
+        <Border x:Name="AttributesBorder" Style="{StaticResource BorderStyle}" Visibility="Collapsed">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto" />
+                    <ColumnDefinition />
+                </Grid.ColumnDefinitions>
+                <Label
+                    Grid.Row="0"
+                    Grid.Column="0"
+                    Grid.ColumnSpan="2"
+                    Content="Edit feature attribute:" />
+                <Label
+                    Grid.Row="1"
+                    Grid.Column="0"
+                    Content="AD_ADDRESS" />
+                <TextBox
+                    x:Name="AdressBox"
```

Comment:
I think XAML files may need reformatting to follow .NET samples style. IIRC x:Name should be on the first line, and grid row/column/span values can be put on one line